LNPopupControllerExample/LNPopupControllerExample/FirstViewController.m

```diff
@@ -122,7 +122,7 @@ - (IBAction)_presentBar:(id)sender
 	targetVC.popupContentView.popupCloseButton.accessibilityHint = NSLocalizedString(@"Custom popup button accessibility hint", @"");
 	
 	targetVC.popupBar.previewingDelegate = self;
-	
+    targetVC.popupBar.progressStyle = LNPopupBarProgressStyleTop;
```

Comment:
This breaks the build. It should also be updated to the new naming convention. Thanks